app/controllers/manageBusinesses/add/IncomeSourceCheckDetailsController.scala

```diff
@@ -150,7 +150,7 @@ class IncomeSourceCheckDetailsController @Inject()(val checkDetailsView: IncomeS
         businessPostalCode = address.postcode,
         businessCountryCode = addIncomeSourceData.countryCode,
         incomeSourcesAccountingMethod = Some(incomeSourcesAccountingMethod),
```

Comment:
incomeSourcesAccountingMethod = addIncomeSourceData.incomeSourcesAccountingMethod,
        cashOrAccrualsFlag = addIncomeSourceData.incomeSourcesAccountingMethod,
        
        And remove `incomeSourcesAccountingMethod <- addIncomeSourceData.incomeSourcesAccountingMethod` line, this would be to prevent making it mandatory